install.sh

```diff
@@ -5,7 +5,7 @@ Hello $(whoami)!
 This Dotfiles plugin installation script will do the following for you:
 - Setup Git repository in you home folder. You will be asked to enter remote address, username, and email
 - Write necessery files:
-    ~/.oh-my-zsh/custom/plugins/dotfiles/dotfiles.plugin.zsh
+    ~ZSH_CUSTOM/plugins/dotfiles/dotfiles.plugin.zsh
```

Comment:
I guess here and everywhere else you should use a dollar sign instead of tilde so that you have `$ZSH_CUSTOM/plugins/dotfiles/dotfiles.plugin.zsh`

Reply:
Hi @vladmyr, yes I think you are right. I had used tilde in my testing but $ is more conventional.